packages/e2e-tests/specs/editor/plugins/__snapshots__/wp-editor-meta-box.test.js.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`WP Editor Meta Boxes Should save the changes 1`] = `"<p>Typing in a metabox</p>"`;
+exports[`WP Editor Meta Boxes Should save the changes 1`] = `"Typing in a metabox"`;
```

Comment:
One of the scripts enqueued in `widgets.php` (which, until this PR, we were loading alongside every block editor) causes TinyMCE to wrap paragraphs with `<p>`. I don't know what it is. I don't _think_ there are any repercussions beyond this test fixture needing to change, but it would be good to double check with some experts AKA @youknowriad and @azaozz.

Reply:
Funny I had the same diff when running the e2e tests against Core a couple weeks ago. Basically in core, I didn't have the `p`s either.

Reply:
Aha, that makes sense as Core never was loading `widgets.php` scripts into the block editor which is what the plugin does.

Reply:
Hi, folks

I had to revert this change in #32915.

Reply:
It looks like after 5.8 beta 3, there wrapping `<p>` is back!